dds/idl/gen_util.h

```diff
@@ -0,0 +1,33 @@
+
+#ifndef OPENDDS_IDL_GEN_UTIL_H
+#define OPENDDS_IDL_GEN_UTIL_H
+
+#include "be_extern.h"
+
+namespace {
+  inline
+  std::string insert_cxx11_accessor_parens(
+                std::string full_var_name, bool is_union_member) {
```

Comment:
Convention is to pass string by const ref.

Reply:
Since full_var_name gets modified within the function a copy makes sense here, does it not?

Reply:
I suppose it gives a performance improvement when c++11 is not being used or if its a union member at the expense of another variable :)